boards/BMS/Src/App/states/App_AllStates.c

```diff
@@ -124,10 +124,13 @@ bool App_AllStatesRunOnTick100Hz(struct StateMachine *const state_machine)
     struct Charger *         charger     = App_BmsWorld_GetCharger(world);
 
     const bool charger_is_connected = App_Charger_IsConnected(charger);
-    bool       status               = true;
+    const bool balancing_enabled    = App_CanRx_Debug_CellBalancing_RequestCellBalancing_Get();
+    const bool ignore_other_boards  = charger_is_connected || balancing_enabled;
```

Comment:
not sure how your charging setup works this year - will LV ever turn on before the BMS detects that the charger is connected, resulting in a premature fault?